mico-admin/src/app/service-detail/service-detail.component.ts

```diff
@@ -74,6 +75,8 @@ export class ServiceDetailComponent implements OnInit, OnDestroy {
                         if (element.version === givenVersion) {
                             this.selectedVersion = givenVersion;
                             this.service = element;
+                            console.log('found version', givenVersion);
```

Comment:
Debug output

Reply:
found variable never set to true! Resulting in always beeing redirected to latest version url.

Reply:
debug output removed.
foreach() replaced by some()